libyui-bindings/CMakeLists.txt

```diff
@@ -14,6 +14,15 @@ endif(COMMAND cmake_policy)
 OPTION( BUILD_RUBY_GEM "Build Ruby GEM" YES )
 OPTION( WITH_MGA "Build with support for libyui-mga" NO )
 
+
+if ( DEFINED ENV{PREFIX} )
+  set( CMAKE_INSTALL_PREFIX $ENV{PREFIX} )
+else()
+  set( CMAKE_INSTALL_PREFIX /usr )
+endif()
+message( "-- CMAKE_INSTALL_PREFIX: ${CMAKE_INSTALL_PREFIX}" )
```

Comment:
Wouldn't it make more sense to not hardwire this at all?

Reply:
The CMake default for this is `/usr/local` which means we, the core developers, always have to remember overriding it.